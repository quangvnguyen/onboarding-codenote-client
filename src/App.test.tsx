import React from 'react';
import { shallow } from 'enzyme';
import RouterApp, { App } from './App';

test('renders without crashing', () => {
  const wrapper = shallow(<RouterApp />);
  expect(wrapper).toMatchSnapshot();
});

test('renders with isAuthenticating', () => {
  const props = {
    userHasAuthenticated: jest.fn(),
  };
  const wrapper = shallow(<App {...props} />);
  wrapper.setState({
    isAuthenticating: false,
  })
  expect(wrapper).toMatchSnapshot();
});