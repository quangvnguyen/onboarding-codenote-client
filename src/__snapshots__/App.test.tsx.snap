// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with isAuthenticating 1`] = `
<div
  className="App container"
>
  <ScreensRoot />
</div>
`;

exports[`renders without crashing 1`] = `
<Route>
  <Component />
</Route>
`;
